lib/ts/recipe/session/types.ts

```diff
@@ -214,12 +235,21 @@ export type RecipeInterface = {
 
     updateSessionData(input: { sessionHandle: string; newSessionData: any; userContext: any }): Promise<void>;
 
+    /**
+     * @deprecated Use mergeIntoAccessTokenPayload instead
+     */
     updateAccessTokenPayload(input: {
         sessionHandle: string;
         newAccessTokenPayload: any;
         userContext: any;
     }): Promise<void>;
 
+    mergeIntoAccessTokenPayload(input: {
```

Comment:
The type of `updateAccessTokenPayload` has changed to return a `Promise<boolean>` in the latest X.Y branch. It returns `false` if the input `sessionHandle` is invalid. Hence, the type of this function should also change.